### PR TITLE
v1.3

### PR DESCRIPTION
**Pull request by MauriceX24**
Oi1kSIFd

### Changelog
| DE | EN |
| ---------------------|-----------------------|
+Freundesliste hinzugefügt | +Added friendship
+Supportdashboard hinzugefügt | +Added supportdashboard
+Support Beschreibung hinzugefügt | +Support Description was added
+Schaut Serie (verbessert) | +Watching a serie (better now)
+Suche verbessert | +Searching is better now
+Anleitung-Link verbessert | +Tutoriallink was changed (fixed)
+serienstream.to entfernt | +Removed serienstream.to
+Abrufen der Links verbessert | +Trigger of links is better then before

### CODE REVIEW

File: s.to/dist/metadata.json [hunk 1]
```diff
@@ -9,20 +9,12 @@
...
         "nl": "Stream meer dan 3500 afleveringen van je favorite serie gratis!" 
     },
-    "url": [
-        "s.to",
-        "board.s.to",
-        "serienstream.to",
-        "serienstream.sx"       
-    ],
-    "version": "1.2",
+    "url": [s.to","board.s.to","serienstream.sx"],
```
Comment (doomlerd) [inline s.to/dist/metadata.json]: You're missing a **"** on the url array.

Comment (MauriceX24) [reply s.to/dist/metadata.json]: Done! :D

**Comment by doomlerd** [comment]
I think you should resolve the conflicting files ( if you can see them )

**Comment by MauriceX24** [comment]
What did you mean? :o

**Comment by doomlerd** [comment]
Click "resolve conflicts" and remove the stuff that you changed on the PR.

**Pull request merged**